src/components/landing/RegisterCard.jsx

```diff
@@ -1,30 +1,59 @@
 import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import LandingCard from './LandingCard.jsx';
 import TextInput from '../TextInput.jsx';
 import Button from '../Button.jsx';
+import API from '../../api/API';
 
 function RegisterCard({ switchCard }) {
-  // eslint-disable-next-line no-unused-vars
+  const history = useHistory();
   const [err, setError] = useState(null);
 
-  const register = event => {
+  async function onSubmit(event) {
     event.preventDefault();
-    setError('Not Implemented');
-  };
+    const data = new FormData(event.target);
+
```

Comment:
Might want to trim the fields so that users can't input strings that are just spaces

Reply:
Got it